packages/ui/components/Layout.tsx

```diff
@@ -1,24 +1,21 @@
-import * as React from 'react';
+import { Container } from '@mui/material';
 import Head from 'next/head';
 import Link from 'next/link';
-
-// FIXME: this should be the layout styles
-import styles from '../styles/Home.module.css';
-
-import { Container } from '@mui/material';
+import * as React from 'react';
 
 import Navbar from '../components/Navbar';
+import styles from './Layout.module.css';
 
 // FIXME: needs to extend from base props correctly
-export const Layout = ({
-  title,
-  children,
-}: {
+interface LayoutProps {
   title: string;
-  children: any;
-}) => {
+  hero?: React.FC;
+  children: JSX.Element | JSX.Element[];
```

Comment:
children does not need to be here, `React.FC` already has children defined it its props

Reply:
looks like definitely typed removed implicit `children` in react-18 defs, we can do this now and it will add proper typing for `children` to the props

```ts
type CustomProps = React.PropsWithChildren<{
  customProp: number
}>
```